src/App.js

```diff
@@ -84,6 +84,30 @@ function App() {
     setEventList(newEventList);
     setCustomEvents((currentList) => [...currentList, newEvent]);
   };
+  const removeEvent = (eventName, startTime, endTime) => {
+    const jsonObj = JSON.parse(localStorage.getItem(customKey)) || [];
+
+    const newCustomList = [];
+    jsonObj.forEach((delEvent) => {
+      console.log(startTime, delEvent.title);
+
+      if (
+        delEvent.title === eventName
+        && delEvent.start === startTime
+        && delEvent.end === endTime
+      ) {
+        return;
+      }
+      newCustomList.push(delEvent);
+    });
```

Comment:
Get rid of the unnecessary console logs. Variables like jsonObj need to be changed to more meaningful names.

Reply:
I agree. Variable names like `delEvent` can be improved. Especially because `delEvent` is actually the event which stays and is *not* being deleted!

A generic name for the function argument like `event` might suffice.